lib/fun_with_flags/ui/router.ex

```diff
@@ -122,9 +122,14 @@ defmodule FunWithFlags.UI.Router do
   # to clear a boolean gate
   #
   delete "/flags/:name/boolean" do
-    flag_name = String.to_existing_atom(name)
-    FunWithFlags.clear(flag_name, boolean: true)
-    redirect_to conn, "/flags/#{name}"
+    case Utils.get_flag(name) do
+      {:ok, flag_name} ->
+        FunWithFlags.clear(flag_name, boolean: true)
```

Comment:
I'm not sure if this would work. `Utils.get_flag(name)` returns `{:ok, flag}` on success, not the flag name. You'd have to use pattern matching to extract the name here.

Reply:
Indeed 😬 I don't remember how I tested it, maybe I did something wrong during my tests. I tried again and it seems to work fine